src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingShepherd.java

```diff
@@ -68,11 +79,54 @@ public AbstractJob createJob(final CitizenData citizen)
         return new JobShepherd(citizen);
     }
 
+    @Override
+    public void serializeToView(@NotNull final ByteBuf buf)
+    {
+        super.serializeToView(buf);
+        buf.writeBoolean(dyeSheeps);
+    }
+
+    @Override
+    public void writeToNBT(@NotNull final NBTTagCompound compound)
+    {
+        super.writeToNBT(compound);
+        compound.setBoolean(NBT_DYE_SHEEPS, this.dyeSheeps);
+    }
+
+    @Override
+    public void readFromNBT(@NotNull final NBTTagCompound compound)
+    {
+        super.readFromNBT(compound);
+        this.dyeSheeps = compound.getBoolean(NBT_DYE_SHEEPS);
```

Comment:
Just missing a if !contains then = true, to make the default true

Reply:
I had it there, then I was like nah, so I removed it

Reply:
=D, I still think that the current default should be the default, else people will wonder why he stopped dying without actively turning it off.

Reply:
done

Reply:
This solution will throw an NPE if NBT_DYE_SHEEPS is not in the compound.
Please surround the compound with the if-switch and use an else switch to handle the default

Reply:
It returns false and (in dev env) doesn't throw an npe. If statement added already